File: BonusServer/Models/BonusRecords.cs
namespace BonusServer.Models
{
    public class BonusRecords
    {
        public List<NotifyData> WinAList { get; set; }
        public List<NotifyData> WinBList { get; set; }
        public List<NotifyData> WinCRList { get; set; }

        public BonusRecords()
        {
            this.WinAList = new List<NotifyData>();
            this.WinBList = new List<NotifyData>();
            this.WinCRList = new List<NotifyData>();
        }

        public BonusRecords Clone()
        {
            BonusRecords newBonusRecords = new BonusRecords();
            //
            newBonusRecords.WinAList = new List<NotifyData>();
            newBonusRecords.WinAList.AddRange(this.WinAList);
            newBonusRecords.WinBList = new List<NotifyData>();
            newBonusRecords.WinBList.AddRange(this.WinBList);
            newBonusRecords.WinCRList = new List<NotifyData>();
            newBonusRecords.WinCRList.AddRange(this.WinCRList);
            return newBonusRecords;
        }
    }
}

File: BonusServer_Node/models/bonusRecords.js

export default class BonusRecords
{
    WinAList;
    WinBList;
    WinCRList;

    constructor()
    {
        this.WinAList = [];
        this.WinBList = [];
        this.WinCRList = [];
    }

    Clone()
    {
        const newBonusRecords = new BonusRecords();
        //
        const contentA = JSON.stringify(this.this.WinAList);
        newBonusRecords.WinAList = JSON.parse(contentA);
        const contentB = JSON.stringify(this.WinBList);
        newBonusRecords.WinBList = JSON.parse(contentB);
        const contentCR = JSON.stringify(this.WinCRList);
        newBonusRecords.WinCRList = JSON.parse(contentCR);
        return newBonusRecords;
    }
}